app/scripts/modules/ecs/ecs.help.ts

```diff
@@ -0,0 +1,24 @@
+import { module } from 'angular';
+import { HELP_CONTENTS_REGISTRY, HelpContentsRegistry } from 'core/help/helpContents.registry';
+
+const helpContents: {[key: string]: string} = {
+  'ecs.loadBalancer.targetGroup': '<p>A <em>target group</em> is attached to an application / network load balancer and are targets for load balancer traffic.</p>  <p> You need to create these resources outside of Spinnaker.</p>',
```

Comment:
The first sentence is a little confusing here:
> A target group is attached to an application/network load balancer and are targets for load balancer traffic.

"A target group" is singular, while "are targets" is plural. Should the sentence start with "Target groups" instead of "A target group"?

Also, which resources need to be created outside of Spinnaker? The target groups themselves?